### PR TITLE
[Bug} Fix Tidy Up buff

### PR DESCRIPTION
**Pull request by Nohbody**
## What are the changes?
1. Fixes Tidy Up buff to only go up 1 stage as intended.

## Why am I doing these changes?
1. Fixes: https://github.com/pagefaultgames/pokerogue/issues/2629

## What did change?
1. Changes the both sides targeting so that the calculation didn't run twice.

## How to test the changes?
overrides.ts: ```export const MOVESET_OVERRIDE: Array<Moves> = [Moves.TIDY_UP];```

## Checklist
- [ ] There is no overlap with another PR?
- [ ] The PR is self-contained and cannot be split into smaller PRs?
- [ ] Have I provided a clear explanation of the changes?
- [ ] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)
- [ ] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/data/move.ts [hunk 1]
```diff
@@ -8280,8 +8280,7 @@ export function initMoves() {
       .target(MoveTarget.BOTH_SIDES),
     new SelfStatusMove(Moves.TIDY_UP, Type.NORMAL, -1, 10, -1, 0, 9)
       .attr(StatChangeAttr, [ BattleStat.ATK, BattleStat.SPD ], 1, true, null, true, true)
-      .attr(RemoveArenaTrapAttr)
-      .target(MoveTarget.BOTH_SIDES),
+      .attr(RemoveArenaTrapAttr, true),
```
Comment (flx-sta) [inline src/data/move.ts:8283]: are you sure you can just remove the `BOTH_SIDES`?

Comment (flx-sta) [inline src/data/move.ts:8283]: removing the `BOTH_SIDES` seems wrong.

> It also clears away [substitutes](https://bulbapedia.bulbagarden.net/wiki/Substitute_(doll)), [Spikes](https://bulbapedia.bulbagarden.net/wiki/Spikes_(move)), [Toxic Spikes](https://bulbapedia.bulbagarden.net/wiki/Toxic_Spikes_(move)), [Stealth Rock](https://bulbapedia.bulbagarden.net/wiki/Stealth_Rock_(move)), and [Sticky Web](https://bulbapedia.bulbagarden.net/wiki/Sticky_Web_(move)) **on both the user's and target's side of the field**

[sauce](https://bulbapedia.bulbagarden.net/wiki/Tidy_Up_(move))

**Review by flx-sta — CHANGES_REQUESTED** [review]

**Comment by flx-sta** [comment]
✅  Tested
✅ code review
✅ tests
lgtm 🚀 

https://github.com/pagefaultgames/pokerogue/assets/50131232/8d93a1d8-daad-4cdd-8647-8eb35a28fb24

**Review by flx-sta — APPROVED** [review]

File: src/data/move.ts (hunk 1 above)
Comment (flx-sta) [reply src/data/move.ts:8283]: Nvm. Both sides is handled inside `RemoveArenaTrapAttr` with the `true` argument (1)

**Review by flx-sta — APPROVED** [review]
✅  Play-Test
✅ code review
✅ tests
lgtm 🚀 

https://github.com/pagefaultgames/pokerogue/assets/50131232/8d93a1d8-daad-4cdd-8647-8eb35a28fb24

_This was after Multiple `Save & Exit` and creating multiple New games with same pokemon etc (as described in #2629)_

**Review by Tempo-anon — APPROVED** [review]
This should undo #2652 and add in the correct fix

**Comment by flx-sta** [comment]
@Nohbody thanks for clarifying this and thanks for the fix

**Pull request merged**